src/main/scala/com/codacy/client/stash/service/PullRequestServices.scala

```diff
@@ -32,30 +38,40 @@ class PullRequestServices(client: StashClient) {
   }
 
   /*
- * Gets the list of comments of a pull request
- *
- */
-  def getPullRequestComments(projectKey: String, repository: String, prId: Long): RequestResponse[Seq[PullRequestComment]] = {
+   * Gets the list of comments of a pull request
+   *
+   */
+  def getPullRequestComments(
+      projectKey: String,
+      repository: String,
+      prId: Long
+  ): RequestResponse[Seq[PullRequestComment]] = {
     val url = s"/rest/api/1.0/projects/$projectKey/repos/$repository/pull-requests/$prId/comments"
 
     client.executePaginated(Request(url, classOf[Seq[PullRequestComment]]))
   }
 
   /*
-  * Comment a file in a pull request
-  */
-  def createComment(projectKey: String, repo: String, prId: Long, body: String, file: Option[String], line: Option[Int], lineType: String = "ADDED"): RequestResponse[PullRequestComment] = {
+   * Comment a file in a pull request
+   */
+  def createComment(
+      projectKey: String,
+      repo: String,
+      prId: Long,
+      body: String,
+      file: Option[String],
+      line: Option[Int],
+      lineType: String = "ADDED"
+  ): RequestResponse[PullRequestComment] = {
     val url = s"/rest/api/1.0/projects/$projectKey/repos/$repo/pull-requests/$prId/comments"
 
-    val params = JsObject(file.map(filename => "path" -> JsString(filename)).toSeq ++
-      line.map(lineTo => "line" -> JsNumber(lineTo)) ++
-      Some("lineType" -> JsString(lineType))
+    val params = JsObject(
```

Comment:
why we are using the lower level API and not mapping this to a `case class` ?

Reply:
This is out of scope for this PR. I am already changing to many things.
I do not want to go into JSON, the changes are just formatting and using the newer play-json apis.

Reply:
fair